src/app/modules/browse/browse.tpl.html

```diff
@@ -74,8 +74,6 @@ <h3 class="bookTitle">{{book.title}}</h3>
                             </li>
                         </ul>
                     </div>
-                    <!-- temporary affordance for trying out the new datagrid. Not the final design -->
-                    <!--form action="#datagrid"><input type="submit" value="Grid view"></form-->
```

Comment:
So long as this is just for admins, it's no problem to have a bookmark to this page. Of course if you want to make a link show up for admins, go for it.